Makefile

```diff
@@ -89,7 +89,7 @@ check-installation:
 	if [ "$(CHECK_BEFORE_INSTALL)" -eq 1 ]; then \
     for x in camlp4 dbm graphics labltk num ocamlbuild; do \
       if [ -f "$(prefix)$(OCAML_SITELIB)/$$x/META" ]; then \
-        if ! grep -Fq '[distributed with Ocaml]' "$(prefix)/$(OCAML_SITELIB)/$$x/META"; then \
+        if ! grep -Fq '[distributed with OCaml]' "$(prefix)/$(OCAML_SITELIB)/$$x/META"; then \
```

Comment:
That will break things...! At the very least it would need to be done with case-insensitive match. Given that OCaml now distributes is own `META` files, I'd be tempted to leave this particular spelling as an ancient artefact 🙂

Reply:
Reverted, indeed a case-insensitive match seems a good first step.